packages/test/test-end-to-end-tests/src/test/gc/gcTombstoneDataStores.spec.ts

```diff
@@ -787,31 +802,46 @@ describeNoCompat("GC data store tombstone tests", (getTestObjectProvider) => {
 					url: unreferencedId,
 					headers: { [AllowTombstoneRequestHeaderKey]: true },
 				};
-				const tombstonedObject = await requestFluidObject<ITestDataObject>(
-					tombstoneContainer,
-					requestAllowingTombstone,
-				);
+				let tombstonedObject: ITestDataObject;
+				{
```

Comment:
Why is this (and similar code below) in a separate block?

Reply:
Was just a style choice to re-use the `response` name. I have no strong opinion one way or the other

Reply:
Me neither :-). I was just curious.